app/src/main/res/layout/tracks_item.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <import type="android.view.View" />
+
+        <variable
+            name="viewHolder"
+            type="org.fossasia.openevent.app.core.track.list.viewholder.TracksViewHolder" />
+
+        <variable
+            name="track"
+            type="org.fossasia.openevent.app.data.tracks.Track" />
+    </data>
+
+    <android.support.v7.widget.CardView
+        android:id="@+id/tracks_card"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:elevation="@dimen/cardview_default_elevation"
+        android:layout_margin="@dimen/spacing_small">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:baselineAligned="false"
+            android:foreground="?selectableItemBackground"
+            android:gravity="center_vertical"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@{ viewHolder.getTrackColor(track) }"
+                android:orientation="vertical"
+                style="@style/ItemPadding">
```

Comment:
This LinearLayout is useless. Instead just have the below header textview take up all the space for this and add background color to that